x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_form.tsx

```diff
@@ -190,21 +192,30 @@ export const AlertForm = ({
 
   const alertTypeNodes = alertTypeRegistryList.map(function (item, index) {
     return (
-      <EuiKeyPadMenuItem
-        key={index}
-        data-test-subj={`${item.id}-SelectOption`}
-        label={item.name}
-        onClick={() => {
-          setAlertProperty('alertTypeId', item.id);
-          setAlertTypeModel(item);
-          setAlertProperty('params', {});
-          if (alertTypesIndex && alertTypesIndex.has(item.id)) {
-            setDefaultActionGroupId(alertTypesIndex.get(item.id)!.defaultActionGroupId);
+      <EuiFlexItem>
+        <EuiCard
+          key={index}
+          data-test-subj={`${item.id}-SelectOption`}
+          layout="horizontal"
+          icon={null}
+          title={item.name}
+          description={
+            <EuiText color="subdued" size="s">
+              Need to add descriptions
```

Comment:
With a dropdown list, probably description doesn't make any sense.

Reply:
I think we should try and keep them if we can.  I'm not sure the names themselves provide enough info for the user.